packages/core/content-manager/server/services/field-sizes.js

```diff
@@ -60,4 +60,26 @@ module.exports = () => ({
 
     return fieldSize;
   },
+  setFieldSize(type, size) {
+    if (!type) {
+      throw new Error('The type is required');
+    }
+
+    if (!size) {
+      throw new Error('The size is required');
```

Comment:
How come these aren't `ApplicationError`? CC @Marc-Roig for guidance on this one

Reply:
Thanks, I didn't know about this convention.